examples/custom_code/model_run.py

```diff
@@ -1,7 +1,6 @@
 """This module loads created ebonite model and then creates and runs docker container made from this model"""
 
 import ebonite
-from ebonite.build.builder.base import use_local_installation
```

Comment:
Please describe motivation to remove it. I'm not against it, I just think that it's better to document it here.

Reply:
this is simple, if you run that code not from ebonite repo, but with pip-installed ebonite, it will fail, cause `setup.py` file from which we load requirements is not there (see ODS ebonite channel)